javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,190 @@
+//! Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+
+if (names.includes(nameToRemove)) {
+  let indexAhmad = names.indexOf(nameToRemove);
```

Comment:
Here you could use const instead of let. Try to look up the difference :)

Reply:
Thank you so much for your feedback!! I appreciate it.